Tasks/AzureCLI/azureclitask.ts

```diff
@@ -198,9 +199,9 @@ export class azureclitask {
         }
     }
 
-    private static createPublishSettingFile(subscriptionName:string, subscriptionId:string, certificate:string, publishSettingFileName:string): void  {
+    private static createPublishSettingFile(subscriptionName:string, subscriptionId:string, certificate:string, serviceManagementUrl:string, publishSettingFileName:string): void  {
         //writing the data to the publishsetting file
```

Comment:
while making a call to azure, the azurecli removes the terminal slash. Thus it is not causing any errors.
here is a screenshot of a call to azure while using a publishsetting file with serviceManagementUrl having terminal slash
![image](https://cloud.githubusercontent.com/assets/20859325/21213201/dcfe5548-c2b8-11e6-946b-49b1966ef39e.png)